modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -1,26 +1,30 @@
 import { Fun } from '@ephox/katamari';
 
+import { expandRng } from 'tinymce/core/fmt/ExpandRange';
+
 import * as CaretRangeFromPoint from '../../selection/CaretRangeFromPoint';
 import * as NormalizeRange from '../../selection/NormalizeRange';
 import * as RangeCompare from '../../selection/RangeCompare';
 import * as RangeNodes from '../../selection/RangeNodes';
 import { RangeLikeObject } from '../../selection/RangeTypes';
 import * as RangeWalk from '../../selection/RangeWalk';
 import * as SplitRange from '../../selection/SplitRange';
-import DOMUtils from './DOMUtils';
+import Editor from '../Editor';
 
 interface RangeUtils {
   walk: (rng: Range, callback: (nodes: Node[]) => void) => void;
   split: (rng: Range) => RangeLikeObject;
   normalize: (rng: Range) => boolean;
+  expand: (range: Range, options?: { type: 'word' }) => Range;
 }
 
 /**
  * This class contains a few utility methods for ranges.
  *
  * @class tinymce.dom.RangeUtils
  */
-const RangeUtils = (dom: DOMUtils): RangeUtils => {
+const RangeUtils = (editor: Editor): RangeUtils => {
```

Comment:
This is a **breaking** change and as such you cannot do this... remember this is a public API

Reply:
What you'll likely need to do to solve this is change `ExpandRange` to take a `DOMUtils` object instead of `Editor` and then change the `editor.getBody()` call inside that to use `dom.getRoot()`.

Reply:
I completely forgot that this was a public API. You'd think I'd remember since it was added here due to this being a public API, but apparently the connection was never made. Fixing that.

Reply:
I really should have spotted this as well since it has a moxiedoc comment above it anytime there is such a comment it's likely an api or if it's in the api directory however that is generally harder to spot since who navigates to files though the file tree these days. I have a tendency to zoom into on the lower levels of the change and I miss these smaller things. Sorry.